Sources/Core/Sources/Network/Protocol/Packets/ClientboundPacket.swift

```diff
@@ -3,12 +3,40 @@ import Foundation
 public enum ClientboundPacketError: LocalizedError {
   case invalidDifficulty
   case invalidGamemode
-  case invalidServerId
+  case invalidServerID
```

Comment:
I was gonna say that I prefer `Id` (which I do because Id is just short for Identifier and it isn't an acronym). But I just checked and Apple does seem to use `ID`, so I guess we can go with that as long as it is updated everywhere that I use `Id` so that it's consistent.

Reply:
Oh, well, you right, acronym is a subset of abbreviation. I guest ID is a little bit special due to wiki https://en.wikipedia.org/wiki/ID and people using it as Id and as ID but I prefer an Apple way with ID.
I saw what you wrote about ID and will revert it to Id